atlp-rwanda/ecommerce-app-champions-fe
# #73 Feature: password change prompt

### what does this PR do?

- user is prompted to change password x amount of time

### description

- implement change password prompt

### how should it be manually tested

-  clone the repo using git clone repo's URL
- change branch git checkout ft-password-change-prompt
- create .env file and paste variables from .env.example
- run npm run dev
- make sure that you are registered 
- navigate to login and paste in your credentials and if your password has expired there will pop up an alert
- click on that alert and navigate to your email 
- check for the password expiry email
- click on it to update your password
